package/MDAnalysis/topology/MMTFParser.py

```diff
@@ -199,9 +199,9 @@ def iter_atoms(field):
         else:
             attrs.append(AltLocs(['']*natoms))
         if len(mtop.b_factor_list):
-            attrs.append(Bfactors(mtop.b_factor_list))
+            attrs.append(Tempfactors(mtop.b_factor_list))
```

Comment:
change line 43? (also a versionchanged or a note somewhere might be good)

Reply:
I thought line 43 already said tempfactors? I'll add a versionchanged